test/integration/targets/filter_urls/runme.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eux
+
+export ANSIBLE_ROLES_PATH=../
+
+ansible-playbook runme.yml "$@"
+
+source virtualenv.sh
+
+# This has to be done outside of requirements.txt but is necessary for
+# installing Jinja 2.6. We need this because Jinja 2.6 won't install with
+# a newer setuptools, and because setuptools 45+ won't work with Python 2.
```

Comment:
I'm curious, what's the issue with Jinja 2.6 on newer setuptools?

Reply:
If you spawn a container (say the fedora31 test container) and activate a venv in the way that `virtualenv.sh` does, and run the pip install, it gives:

```
    ImportError: cannot import name 'Feature' from 'setuptools' (/root/ansible/test/results/.tmp/output_dir/venv/lib/python3.7/site-packages/setuptools/__init__.py)
```

I looked upstream (setuptools) and found that this breakage is expected due to API changes they made.